client/components/main/About.jsx

```diff
@@ -64,30 +65,70 @@ const About = () => {
               the expansive amount of data associated with these 311
               requests is available online. The mayor has encouraged
               us to create apps with this data, and that&apos;s where
-              this project comes in.
+              this project comes&nbsp;in.
```

Comment:
can all instances of `&nbsp;` be replaced with a single space?

Reply:
Not sure what you mean? non breaking space here is to prevent the orphan text.

Reply:
i see. the reason just wasn't clear. but if you find it important to keep those words together this will definitely work